project_status/models/project_status.py

```diff
@@ -0,0 +1,20 @@
+from odoo import models, fields, api
+
+
+class ProjectStatus(models.Model):
+    _name = 'project.status'
+    _order = 'status_sequence'
+
+    name = fields.Char(string="Name",
+                       required=True)
+    description = fields.Char(string="Description")
+    status_sequence = fields.Integer(string="Sequence")
+    is_closed = fields.Boolean(string="Is Closed Status",
+                               help="Specify if this is a closing status.")
+    fold = fields.Boolean(string="Folded")
+
+    @api.model
+    def create(self, vals):
+        seq = self.env['ir.sequence'].next_by_code('project.status') or '/'
+        vals['status_sequence'] = seq
```

Comment:
assigning '/' to the `status_sequence` Integer field can result in errors.

Reply:
Thanks, i've removed the or '/'

Reply:
In another module you reviewed, you suggested instead of or '/' to replace with or 0, so i did the same with this module.